Apsitvarkom.Models/PollutedLocation.cs

```diff
@@ -32,4 +32,7 @@ public enum SeverityLevel
 
     /// <summary>Geographic location information about the record.</summary>
     public Location Location { get; set; } = new();
+
+    /// <summary>Collection of events linked to the location.</summary>
+    public List<TidyingEvent> Events { get; set; } = new();
```

Comment:
Apparently, DB doesn't return any data for this `Events` property, as it is (logically, due to unwanted data repetitions) not stored in the `PollutedLocation` table. This is why I didn't include it in the `PollutedLocationResponse`. Will play around and see if we could possibly return it anyways, as I believe it would be beneficial when listing all the events in the sidebar as well.

Reply:
resolved in `PollutedLocationDatabaseRepository`, where I had to specify that I'd like to include `Events` into the returned `PollutedLocation` instance(s).